### ISSUE TITLE
bad numeric position

### ISSUE THREAD

**Issue by pbranly** [nemethviktor/GeoTagNinja#15]
Hi
When I select a position, it is good in the popup  but latitude longitude is wrong
see screen copy
Si it write the bad references in the photos
Phil
![2022-11-19_15-24](https://user-images.githubusercontent.com/22018509/202855715-c2d93123-30ff-46be-934f-f090810aabf2.png)

**Comment by nemethviktor** [comment]
Ehh. Yeah that might be after the refactoring - I'll have a look next week
The previous version should be ok - apologies

This message has been brought to you by my Huawei P30 Pro and myself. Any
typos blame the phone.

**Comment by nemethviktor** [comment]
Try this one https://drive.google.com/file/d/18iI77SIdrIv-joOtyT0-MzqOVtB5OgM0/view?usp=share_link

Annoyingly I s...t you not that was a 3-character code change. I unleashed some automatic thing on the code the other day to make it more readable and standardised and it introduced a few things I didn't want. I only tested it in the non-English VMs to the extent that the app works but didn't check the map-click. My bad.
